tpp-app/src/services/utils/helpers.js

```diff
@@ -16,3 +18,29 @@ const initializeEmptyYear = (yearNumber) => {
 
   return year
 }
+
+
+/**
+ * Convert a Date object into a date string, encoding year, month and day. Note it encodes months as 1 indexed, and days as 0 indexed
+ * @param {Date} date Object to convert to string
+ * @return {String} String encoding year, month and day
+ */
+export const getDateString = (date) => {
+  var date = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate();
+  return date;
+
+}
+
+/**
+ * Retrieves the user's symptom data for the given date.
+ * @param day number
+ * @param month number (January = 0)
```

Comment:
I think for arguments, we would pass 1 for January

Reply:
oh yeah u right